diff --git a/node_modules/react-native-wechat-lib/android/build.gradle b/node_modules/react-native-wechat-lib/android/build.gradle
index ed9f33b..5b0b4b3 100644
--- a/node_modules/react-native-wechat-lib/android/build.gradle
+++ b/node_modules/react-native-wechat-lib/android/build.gradle
@@ -15,9 +15,10 @@ def isNewArchitectureEnabled() {
 
 apply plugin: "com.android.library"
 
-if (isNewArchitectureEnabled()) {
-  apply plugin: "com.facebook.react"
-}
+// Plugin disabled for React Native 0.70 compatibility
+// if (isNewArchitectureEnabled()) {
+//   apply plugin: "com.facebook.react"
+// }
 
 def getExtOrDefault(name) {
   return rootProject.ext.has(name) ? rootProject.ext.get(name) : project.properties["WechatLib_" + name]
@@ -30,7 +31,7 @@ def getExtOrIntegerDefault(name) {
 
 android {
-  compileSdkVersion getExtOrIntegerDefault("compileSdkVersion")
+  compileSdkVersion getExtOrIntegerDefault("compileSdkVersion") ?: 31
 
   defaultConfig {
     minSdkVersion getExtOrIntegerDefault("minSdkVersion")
@@ -63,16 +64,17 @@ repositories {
 dependencies {
   // For < 0.71, this will be from the local maven repo
   // For > 0.71, this will be replaced by `com.facebook.react:react-android:$version` by react gradle plugin
   //noinspection GradleDynamicVersion
-  implementation "com.facebook.react:react-native"
+  implementation "com.facebook.react:react-native:0.70.0"
   api group: 'com.tencent.mm.opensdk', name: 'wechat-sdk-android', version: '6.8.20'
 }
 
-if (isNewArchitectureEnabled()) {
-  react {
-    jsRootDir = file("../src/")
-    libraryName = "WechatLib"
-    codegenJavaPackageName = "com.wechatlib"
-  }
-}
+// Disabled for React Native 0.70 compatibility
+// if (isNewArchitectureEnabled()) {
+//   react {
+//     jsRootDir = file("../src/")
+//     libraryName = "WechatLib"
+//     codegenJavaPackageName = "com.wechatlib"
+//   }
+// }
